lock.go

```diff
@@ -0,0 +1,244 @@
+package ibclient
+
+import (
+	"fmt"
+	"github.com/Sirupsen/logrus"
+	"math/rand"
+	"time"
+)
+
+const (
+	timeout     int32  = 60 // in seconds
+	freeLockVal string = "Available"
+)
+
+type Lock interface {
+	Lock() bool
+	UnLock(force bool) bool
+}
+
+type NVLocker struct {
```

Comment:
Just to be consistent, with the naming of the types the package, you might want to spell out NetworkView - i.e. NetworkViewLocker() instead of NVLocker().